lib/config/common-media-library.api.md

```diff
@@ -832,6 +835,9 @@ export type DecodedId3Frame<T> = {
 // @beta
 export function decodeFairPlayLicense(response: string | ArrayBuffer): Uint8Array;
 
+// @beta
+export function decodeHex(hex: string): ArrayBuffer;
```

Comment:
Is it worth adding 
```suggestion
export function decodeHexToArrayBuffer(hex: string): ArrayBuffer;
```

It will be useful if you will have something else (e.g. Int8Array) in the future and you won't need to rename it and deprecation.

Same below for other functions.

What do you think?